test/datastream/datastream_test.go

```diff
@@ -137,22 +137,17 @@ var _ = Describe("Streaming Data Conversion", func() {
 
 func getImageVirtualSize (outFile string) int64 {
 	//call qemu-img info
-	args1 := []string{"info", outFile}
 
 	//create command
-	//TODO: future investigate why this doesn't work
-	// 	cmd := fmt.Sprintf("qemu-img info %s | grep 'virtual size:'", outFile)
-	//  out, err := exec.Command("bash","-c",cmd).Output()
-	out, err := exec.Command("qemu-img", args1...).Output()
+	cmd := fmt.Sprintf("qemu-img info %s | grep 'virtual size:'", outFile)
+	out, err := exec.Command("sh","-c",cmd).Output()
 	if err != nil {
 		return 0
 	}
 	sOut := string(out)
 
 	index1 := strings.Index(sOut, "(")
-	sPart1 := sOut[index1+1:len(sOut)]
-	index2 := strings.Index(sPart1, " ")
-	sSize := sPart1[0:index2]
+	sSize := sOut[index1+1:len(sOut)-8]
```

Comment:
Can this 8 be a const or var defining the len of a string you're excluding?

Reply:
sure